ide/templates/index.html

```diff
@@ -26,7 +26,7 @@
   </script>
 
 </head>
-<body>
+<body style="background: #F3F5F7;">
```

Comment:
don't add style tags here maintain them in css

Reply:
Oh okay!